puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
When I say "I think the right philosophy to have" there, you should imagine me shrugging my shoulders and sounding uncertain.